openc3/lib/openc3/utilities/store_autoload.rb

```diff
@@ -21,6 +21,7 @@
 # if purchased from OpenC3, Inc.
 
 require 'redis'
+require "hiredis-client"
```

Comment:
I love this comment in https://github.com/redis-rb/redis-client?tab=readme-ov-file#drivers:
`For increased performance, you can enable the hiredis binding by adding hiredis-client to your Gemfile`
Like, why don't you just detect I'm not on Windows and do this for me?!